modules/vaos/app/services/ccra/referral_service.rb

```diff
@@ -13,11 +13,13 @@ class ReferralService < BaseService
     def get_vaos_referral_list(icn, referral_status)
       data = { ICN: icn, ReferralStatus: referral_status }
       with_monitoring do
+        # Skip token authentication for mock requests
+        req_headers = config.mock_enabled? ? {} : headers
```

Comment:
Note: Why this? Bypassing the token authentication gets rid of the need to have a valid RSA key somewhere in the repo for the JWT wrapper. The RSA key can be considered a secret even if it is fake.